src/lib/modules/SearchControls/SearchError.js

```diff
@@ -1,21 +1,48 @@
+import React, { Component } from 'react';
 import { Header, Icon, Segment } from 'semantic-ui-react';
-import React from 'react';
 import PropTypes from 'prop-types';
+import _get from 'lodash/get';
 
-export const SearchError = ({ error }) => {
-  return (
-    <Segment placeholder textAlign="center">
-      <Header icon>
-        <Icon name="warning" />
-        Oups! Something went wrong while fetching results.
-      </Header>
-      <Header.Subheader>
-        Please try again later. If the problem persists, contact the technical
-        support.
-      </Header.Subheader>
-    </Segment>
-  );
-};
+export class SearchError extends Component {
+  renderInvalidQueryError = () => {
+    return (
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="question" />
+          Unrecognized search query.
+        </Header>
+        <Header.Subheader>
+          The query you entered was not recognized. Please refer to the
+          documentation for more information.
+        </Header.Subheader>
+      </Segment>
+    );
+  };
+
+  renderOtherError = () => {
+    return (
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="warning" />
+          Something went wrong while fetching results.
+        </Header>
+        <Header.Subheader>
+          Please try again later. If the problem persists, contact the technical
+          support.
```

Comment:
Can we add the contact? E-mail? It should be in the `invenioConfig.APP`. Double check other places how we display it.